serving/samples/helloworld-java/README.md

```diff
@@ -200,11 +208,12 @@ folder) you're ready to build and deploy the sample app.
    helloworld-java     helloworld-java.default.example.com
    ```
 
-1. Now you can make a request to your app to see the result. Replace
-   `{IP_ADDRESS}` with the address you see returned in the previous step.
+1. Now you can make a request to your app to see the result. Presuming,
+   the IP address you got in the step above is in the `${IP_ADDRESS}`
+   env variable:
 
    ```shell
-   curl -H "Host: helloworld-java.default.example.com" http://{IP_ADDRESS}
+   curl -H "Host: helloworld-java.default.example.com" http://${IP_ADDRESS}
```

Comment:
nit: The domain may not be example.com

It would be better to get it. See https://github.com/knative/docs/tree/master/serving/samples/rest-api-go#access-the-service

Reply:
And done.